BillingApp/user/views.py

```diff
@@ -1,6 +1,11 @@
 from django.contrib.auth.views import LoginView as AuthLoginView
 from django.shortcuts import render
-
+from django.http import HttpResponseRedirect
+from django.contrib.auth import logout
 
 class LoginView(AuthLoginView):
     template_name = 'registration/login.html'
+
+def Logout(request):
```

Comment:
maybe we could use [`LogoutView`](https://docs.djangoproject.com/en/3.0/topics/auth/default/#django.contrib.auth.views.LogoutView)